### PR TITLE
git-shortlog: add page

### PR DESCRIPTION
**Pull request by thescalaguy**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/git-shortlog.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- View a summary of all the commits made, grouped alphabetically by author name:
+
+`git shortlog`
+
+- View a summary of all the commits made, sorted by the number of commits made:
+
+`git shortlog -n`
+
+- View a summary of all the commits made, grouped by the commiter identities:
```
Comment (agnivade) [inline pages/common/git-shortlog.md]: What does `commiter identities` mean ? Is it just name ?

Comment (agnivade) [reply pages/common/git-shortlog.md]: I couldn't even find this option in the man page. Can you paste the text here ?

Comment (pxgamer) [reply pages/common/git-shortlog.md]: `-c / --committer`
>Collect and show committer identities instead of authors.

Comment (agnivade) [reply pages/common/git-shortlog.md]: Yes, it does seem to be email and name. Let's change the description to reflect that.

Comment (thescalaguy) [reply pages/common/git-shortlog.md]: @agnivade Done. I've kept "committer identities" in the description to make the man pages easy to understand if someone goes from TLDR to man

**Pull request merged**